apis/stardust/types/index.d.ts

```diff
@@ -153,6 +153,13 @@ export function useOptions(): object;
  */
 export function onTakeSnapshot(snapshotCallback: ($: EngineAPI.IGenericObjectLayout)=>Promise<EngineAPI.IGenericObjectLayout>): void;
 
+/**
+ * Registers a callback that is called when the context menu opens
+ * @param addItemCallback
+ */
+ export function onContextMenu(addItemCallback: (menu: any, event: any) => void): void;
+
+ 

```

Comment:
In my opinion this file should only be the unchanged auto generated output.
meaning onContextMenu should not be included as long as it is marked by `@ignore`

Reply:
I don't know how it is still here, I reverted my last commit ! I'll remove it.